blocks/header/header.css

```diff
@@ -1,275 +1,459 @@
-/* header and nav layout */
-header .nav-wrapper {
-  background-color: var(--background-color);
-  width: 100%;
-  z-index: 2;
-  position: fixed;
+/*
+Cutpoint 1: 768px - hide hamburger
+
+*/
+
+:root {
```

Comment:
should this go to styles.css?

Reply:
Yes, for now trying to keep common css under :root of this block, eventually will move these to a common place like styles.css

Reply:
Will be taken care in #33 PR